### DISCUSSION TITLE
Why the kong issue 9016 is not available in the internet

### DISCUSSION THREAD

**Discussion by gruummy** [Kong/kong#11232] [Help] [answered]
We faced on your side this issue in the past and thanks to the description we had been able to workaround it in our custom lua plugin.

Now we made an error again in our custom lua plugin on our side and wanted to review the issue again.
... But is is gone. !!!!!

Why this happens and how i can access this issue description and history again ?

**Comment by StarlightIbuki** [comment, marked as answer]
Hi. Sorry for the inconvenience. The issue is security-related so it's made unpublic. It's already fixed (from both Kong and OpenResty). Upgrading to the latest version is recommended.

**Reply by wangxuefeismart** [reply]
yes, it is because the anti-fire wall
